aspnetcore/fundamentals/servers/kestrel/http3.md

```diff
@@ -46,7 +49,9 @@ The preceding Windows 11 Build versions may require the use of a [Windows Inside
 
 ### Linux
 
-On Linux, `libmsquic` is published via Microsoft official Linux package repository `packages.microsoft.com`. In order to consume it, it must be added manually. See [Linux Software Repository for Microsoft Products](/windows-server/administration/linux-package-repository-for-microsoft-software). After adding `libmsquic`, it can be installed via the package manager of your distro, for example, for Ubuntu:
+* `libmsquic` package installed.
+
+`libmsquic` is published via Microsoft official Linux package repository `packages.microsoft.com`. In order to consume it, it must be added manually. See [Linux Software Repository for Microsoft Products](/windows-server/administration/linux-package-repository-for-microsoft-software). After adding `libmsquic`, it can be installed via the package manager of your distro, for example, for Ubuntu:
```

Comment:
@wtgodbe @Tratcher @adityamandaleeka I've read this paragraph a number of times and I honestly can't make sense of what it is saying.

> In order to consume it, it must be added manually.

What is it? The repository or the package? Or does the first "it" refer to the package and the second "it" refer to the package repository? 🤷 

> After adding `libmsquic`, it can be installed via the package manager of your distro, for example, for Ubuntu:

Adding libmsquic is different from installing it? Or is this sentence incorrect and after adding the feed, the package can be installed.

Reply:
Yea, that's quite unclear. Let's rephrase.

>  Or is this sentence incorrect and should mean that after adding the feed, the package can be installed?

^ That's right